modules/masorange_rules_helm/1.7.0/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+bcr_test_module:
+  module_path: examples
+  matrix:
+    platform:
+    - debian10
+    - ubuntu2004
+    - macos
+    - macos_arm64
+    bazel:
+    - 7.x
```

Comment:
did you mean to stop testing on 8.x? the previous version did test on 8.x

Reply:
hi @Wyverald!
Yes, that was what I wanted to do in my las commit. But I've already discovered what was causing failures on bazel 8.x builds. I'm gonna close this PR and generate a new release. 
Thanks!